src/plugins/home/public/application/components/home.js

```diff
@@ -116,105 +113,144 @@ export class Home extends Component {
+                    </EuiFlexItem>
+                  ) : null}
+
+                  {devTools ? (
+                    <EuiFlexItem className="homHeader__actionItem">
+                      <EuiButtonEmpty
+                        onClick={createAppNavigationHandler(devTools.path)}
+                        iconType="wrench"
+                      >
+                        {i18n.translate('home.pageHeader.devToolsButtonLabel', {
+                          defaultMessage: 'Dev tools',
+                        })}
+                      </EuiButtonEmpty>
+                    </EuiFlexItem>
+                  ) : null}
+                </EuiFlexGroup>
+              </EuiFlexItem>
+            </EuiFlexGroup>
+          </div>
+        </header>
+
+        <div className="homContent">
+          {solutions.length && <SolutionsSection addBasePath={addBasePath} solutions={solutions} />}
+
+          <EuiFlexGroup
+            className={`homData ${
+              addDataFeatures.length === 1 && manageDataFeatures.length === 1
+                ? 'homData--compressed'
+                : 'homData--expanded'
+            }`}
+          >
+            <EuiFlexItem>
+              <AddData addBasePath={addBasePath} features={addDataFeatures} />
             </EuiFlexItem>
+
             <EuiFlexItem>
-              <EuiPanel paddingSize="l">
-                <EuiTitle size="s">
-                  <h2>
-                    <FormattedMessage
-                      id="home.directories.manage.nameTitle"
-                      defaultMessage="Manage and Administer the Elastic Stack"
-                    />
-                  </h2>
-                </EuiTitle>
-                <EuiSpacer size="m" />
-                <EuiFlexGrid columns={2}>
-                  {this.renderDirectories(FeatureCatalogueCategory.ADMIN)}
-                </EuiFlexGrid>
-              </EuiPanel>
+              <ManageData addBasePath={addBasePath} features={manageDataFeatures} />
             </EuiFlexItem>
           </EuiFlexGroup>
 
-          <EuiSpacer size="l" />
+          <EuiHorizontalRule margin="xl" aria-hidden="true" />
 
-          <EuiFlexGroup justifyContent="center">
-            <EuiFlexItem grow={false} className="eui-textCenter">
-              <EuiText size="s" color="subdued">
-                <p>
+          <footer className="homFooter">
+            <EuiFlexGroup alignItems="center" justifyContent="spaceBetween">
+              <EuiFlexItem grow={false}>
+                {advancedSettings && (
+                  <EuiButtonEmpty
+                    iconType="home"
+                    onClick={createAppNavigationHandler(
+                      '/app/management/kibana/settings#defaultRoute'
+                    )}
+                    size="xs"
+                  >
+                    <FormattedMessage
+                      id="home.changeHomeRouteLink"
+                      defaultMessage="Display a different page on log in"
```

Comment:
```suggestion
                      defaultMessage="Change the home page for this space"
```
Small suggestion in wording.

Feels a little more accurate and is an active action.

@gchaps @MichaelMarcialis

Reply:
There were many discussions for this particular piece of copy, all of which included Gail ;) 

This feature doesn't actually change your home page, per se, as it does your start page. In other words, clicking the home button will take you to the Kibana home page even if you set a new route. Additionally, we had to account for situation where Spaces is/is not enabled. Given all those factors, we arrived at this copy.

Reply:
Sounds good. Thanks for the clarity. Resolved.